gammapy/estimators/map/core.py

```diff
@@ -1093,6 +1094,31 @@ def read(cls, filename, checksum=False):
         ) as hdulist:
             return cls.from_hdulist(hdulist, checksum=checksum)
 
+    def copy(self, reference_model=None):
+        """Deep copy
+
+        Parameters
+        ----------
+
+        reference_model : `~gammapy.modeling.models.SkyModel`, optional
```

Comment:
If with a new reference model the fluxes are not modified, what is the use case at all?

Reply:
the use case came from here : was suggested here
https://github.com/gammapy/gammapy/pull/5275#discussion_r1621864916

Reply:
Thanks, that make sense! I would maybe just mention the EBL absorption as a valid use case.